press/views/api-docs/swagger.yaml

```diff
@@ -702,6 +705,29 @@ paths:
           type: file
           description: The litezip file containing the content
           required: true
+  '/api/collections/{id}/{ver}':
```

Comment:
Rather than build API that is deprecated from birth, 
how about using the existing API on archive, from neb:
```
>>> import requests
>>> r = requests.get('https://archive.cnx.org/content/col11995/latest')
>>> col = r.json()
>>> col['legacy_version']
u'1.16'
```
followed by:
```
>>> r2 = requests.get('https://archive.cnx.org/extras/{}'.format(col['id']))
>>> colx = r2.json()
>>> colx['isLatest']
True
>>>colx['downloads']
...
```
And so on.

Reply:
This suffers the same 'state' problem you mentioned above.

Reply:
Let's amend a new special term for `ver`. `head` in addition to `latest`. I'll code that up...

Reply:
To be clear, are you coding that here, or in archive?

Reply:
Here. The PR has been updated.